### PR TITLE
[DOC] Update tempo-distributed helm chart doc

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:

Update Helm chart procedures for accuracy.

New: Large configuration files examples are now collapsed using the collapse shortcode. 

image

**Which issue(s) this PR fixes**:
Fixes [these issues](https://github.com/grafana/tempo/issues/4326#issuecomment-2484268069) in https://github.com/grafana/tempo/issues/4326

**Checklist**
- [ ] Tests updated
- [X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md [hunk 1]
```diff
@@ -322,39 +368,32 @@ If you don't need to change the values, you can skip this section.
...
-Refer to the [`storage` configuration block]({{< relref "/docs/tempo/latest/configuration#storage" >}}) for information on storage options.
-
-To use other storage options, set `minio.enabled: false` in the `values.yaml` file:
-
-```yaml
----
-minio:
-  enabled: false # Disables the MinIO chart
-```
+Refer to the [`storage` configuration block](httpe://docs/tempo/<TEMPO_VERSION>/configuration#storage) for information on storage options.
```
Comment (knylander-grafana) [inline docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md:379]:
```suggestion
Refer to the [`storage` configuration block](https://grafana.com/docs/tempo/<TEMPO_VERSION>/configuration#storage) for information on storage options.
```

**Review by ie-pham — APPROVED** [review]

File: docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md [hunk 2]
```diff
@@ -266,7 +311,8 @@ Before you run the Helm chart, you need to configure where to store trace data.
 The `storage` block defined in the `values.yaml` file configures the storage that Tempo uses for trace storage.
 
 The procedure below configures MinIO as the local storage option managed by the Helm chart.
-However, you can use a other storage provides. Refer to the Optional storage section below.
+However, you can use a other storage provides.
```
Comment (ayah-el) [inline docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md:314]:
```suggestion
However, you can use another storage provider.
```

File: docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md [hunk 3]
```diff
@@ -367,19 +406,14 @@ traces:
...
-    enabled: false
 ```
 
+For GET, the Enterprise Gateway is enabled by default, which only receives traces in OTLP gRPC and HTTP protocol.
+
 ### Optional: Add custom configurations
 
 There are many configuration options available in the `tempo-distributed` Helm chart.
-This procedure only covers the bare minimum required to launch GET or Tempo in a basic deployment.
+This procedure only covers the minimum required to launch GET or Tempo in a basic deployment.
```
Comment (ayah-el) [inline docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md:416]:
```suggestion
This procedure only covers the minimum configuration required to launch GET or Tempo in a basic deployment.
```

File: docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md [hunk 4]
```diff
@@ -401,7 +435,7 @@ Replace `<ingress-host>` with a suitable hostname that DNS can resolve to the ex
 For more information, refer to [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/).
 
 {{< admonition type="note" >}}
-On Linux systems, and if it's not possible for you set up local DNS resolution, you can use the `--add-host=<ingress-host>:<kubernetes-cluster-external-address>` command-line flag to define the `<ingress-host>` local address for the docker commands in the examples that follow.
+If you are a Linux systems and it's not possible for you set up local DNS resolution, use the `--add-host=<ingress-host>:<kubernetes-cluster-external-address>` command-line flag to define the `<ingress-host>` local address for the Docker commands in the examples that follow.
```
Comment (ayah-el) [inline docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md:438]:
```suggestion
If you are using a Linux system and it's not possible for you set up local DNS resolution, use the `--add-host=<ingress-host>:<kubernetes-cluster-external-address>` command-line flag to define the `<ingress-host>` local address for the Docker commands in the examples that follow.
```

**Review by ayah-el — CHANGES_REQUESTED** [review]
A few suggested fixes

File: docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md (hunk 4 above)
Comment (knylander-grafana) [reply docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md:438]: Very good catch on this.

**Comment by knylander-grafana** [comment]
> A few suggested fixes

Done. would you review again?

**Review by ayah-el — APPROVED** [review]
lgtm ✅

**Pull request merged**